src/pages/Admin/VenueOwnersModal.tsx

```diff
@@ -80,23 +115,15 @@ export const VenueOwnersModal: React.FC<VenueOwnersModalProps> = (props) => {
             onChange={(e) => debouncedSearch(e.target.value)}
           />
           <div className="row-container">
-            {typeof filteredUsers === "undefined" ? (
+            {hasResults &&
+              (filteredUsers ?? []).map((user) => (
```

Comment:
`filteredUsers` should always be defined when `hasResults` is, but TypeScript didn't seem to agree. This was a bit of a hacky workaround for that.

Reply:
yep, its type inference isn't perfect, so no prob at all.